etc/upgrades/1705556759-fee-model-and-1.4.1/stage2/crypto.json

```diff
@@ -0,0 +1,11 @@
+{
+  "verifier": {
+    "address": "0x3390051435eCB25a9610A1cF17d1BA0a228A0560",
+    "txHash": "0x2c3b8915e8ebc617cc13e462df334f4197d48b19b302538a519f55acbda285cb"
+  },
+  "keys": {
+    "recursionNodeLevelVkHash": "0x5a3ef282b21e12fe1f4438e5bb158fc5060b160559c5158c6389d62d9fe3d080",
+    "recursionLeafLevelVkHash": "0x062362cb3eaf1f631406cbe19bf2a2c5d0d9ea69d069309a6003addae9f387be",
+    "recursionCircuitsSetVksHash": "0xf12256f7ba1ab223ba82abb19c8366c6677596897f9e5a82016977113e52ed5d"
```

Comment:
`recursionCircuitsSetVksHash` should be zero. Please also update tool to set it to zero always

Reply:
https://github.com/matter-labs/zksync-era/blob/sb-short-term-fee-model-1-4-1/etc/env/base/contracts.toml#L43 - shouldn't it be taken from here? Or should we update it as well?

Reply:
No, it's but that it reads this variable